zebrad/src/application.rs

```diff
@@ -159,7 +159,13 @@ impl Application for ZebradApp {
             .add_issue_metadata("version", env!("CARGO_PKG_VERSION"))
             .add_issue_metadata("git commit", Self::git_commit())
             .issue_filter(|kind| match kind {
-                color_eyre::ErrorKind::NonRecoverable(_) => true,
+                color_eyre::ErrorKind::NonRecoverable(error) => {
+                    let error_str = match error.downcast_ref::<String>() {
+                        Some(as_string) => as_string,
+                        None => return true,
+                    };
+                    !error_str.contains("already in use")
```

Comment:
We only want to skip bug reports for expected port and database errors.

If we get a different kind of error in the same place, we want a bug report. So let's filter based on the strings in the errors (not the strings we added to the panics).

Windows and unix-based platforms both say "already in use", see #1484 and https://support.microsoft.com/en-us/help/261194/you-receive-an-error-message-when-you-try-to-start-the-web-service

RocksDB returns an error containing the words "lock file", and either "temporarily unavailable" on unix (#1508) or "in use" on Windows: https://helpcenter.gsx.com/hc/en-us/articles/115015880627-How-to-Identify-which-Windows-Process-is-Locking-a-File-or-Folder

```suggestion
                    // listener port conflicts
                    if error_str.contains("already in use") {
                        return false;
                    }
                    // RocksDB lock file conflicts
                    if error_str.contains("lock file") && (error_str.contains("temporarily unavailable") || error_str.contains("in use")) {
                        return false;
                    }
                    true
```